db/migrate/20231228141153_add_description_to_collections.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddDescriptionToCollections < ActiveRecord::Migration[7.1]
+  def change
+    add_column :collections, :description, :string, default: '', null: false
```

Comment:
I thought it would be convenient to always have a valid string as the description.

Reply:
That's true, and the `NULL` value might not persist easily: Consider someone editing (or creating) a new collection. When submitting the HTML form, a user will always submit a value, even if empty. Hence, a regular SQL `UPDATE` will quickly set the value to `''`, since `NULL` cannot be set in the same way. Therefore, migrating to an empty string is also fine for me.